lib/ansiblelint/__init__.py

```diff
@@ -196,7 +196,9 @@ def listtags(self):
     @classmethod
     def create_from_directory(cls, rulesdir):
         result = cls()
-        result.rules = ansiblelint.utils.load_plugins(os.path.expanduser(rulesdir))
+        rulesdir = os.path.expanduser(rulesdir)
+        rulesdir = os.path.expanduser(rulesdir)
```

Comment:
Typo during copy-paste:
```suggestion
        rulesdir = os.path.expandvars(rulesdir)
```